### PR TITLE
Insta user description

### PR DESCRIPTION
**Pull request by Tarush27**
how to manage the margin between user_description_two (text view) and bell icon drawable (image view).

### CODE REVIEW

**Comment by abhaysood** [comment]
@Tarush27 attach a screenshot of what the UI looks like with what you have done so far.

**Comment by Tarush27** [comment]
@abhaysood I have added a screenshot of my UI for your reference.
dribble-ui

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -114,10 +114,34 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/insta_user_description_one" />
+
+    <ImageView
+        android:id="@+id/bell_icon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/insta_user_description_two"
+        android:layout_marginStart="25dp"
```
Comment (abhaysood) [inline app/src/main/res/layout/activity_main.xml:143]: You have a 25dp margin already, what do you want to change?

Comment (Tarush27) [reply app/src/main/res/layout/activity_main.xml:143]: I don't want to use an image view for this bell icon drawable. There is an attribute in the text view known as drawable bottom through which we can place our drawable directly below the text view. But this does not happen as the drawable align at the center. Even, the android:gravity attribute is not working.

Comment (abhaysood) [reply app/src/main/res/layout/activity_main.xml:143]: Have you tried 
https://developer.android.com/reference/android/widget/TextView#attr_android:drawablePadding

If this doesn't work then look at:
 https://developer.android.com/reference/android/widget/TextView#setCompoundDrawablePadding(int)

Comment (Tarush27) [reply app/src/main/res/layout/activity_main.xml:143]: I think it's not possible to align the bell icon drawable to the bottom left position inside the text view. I saw these links but again they tell me about drawablePadding which is not required.

**Comment by abhaysood** [comment]
Atach the screenshot of the original design you're trying to replicate.
Add it to the PR description.

On January 30, 2022, Github Notifications ***@***.***>
wrote:
> @Tarush27 commented on this pull request.
>

> In app/src/main/res/layout/activity_main.xml
> <https://github.com/Tarush27/Dribble-UI-
> Demo/pull/4#discussion_r795294180>:
>

> + android:layout_height="wrap_content" +
***@***.***/horizontal_line_divider" /> + + <TextView +
android:id="@+id/insta_user_description_two" +
***@***.***/InstaUserDescriptionTwo" +
android:layout_width="wrap_content" +
android:layout_height="wrap_content" +
***@***.***/insta_user_description_one" /> + + <ImageView
+ android:id="@+id/bell_icon" + android:layout_width="wrap_content" +
android:layout_height="wrap_content" +
***@***.***/insta_user_description_two" +
android:layout_marginStart="25dp"
> I think it's not possible to align the bell icon drawable to the
> bottom left position inside the text view. I saw these links but again
> they tell me about drawablePadding which is not required.
>

—
Reply to this email directly, view it on GitHub
<https://github.com/Tarush27/Dribble-UI-
Demo/pull/4#discussion_r795294180>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AA32JIZZYHQGATSR2UKCWWLUYXR7XANCNFSM5NALWNPA>.
You are receiving this because you were mentioned.Message ID:
***@***.***>

**Comment by Tarush27** [comment]
@abhaysood I have attached the screenshot of my UI. 
Screenshot from 2022-01-31 12-03-23

**Comment by abhaysood** [comment]
Asked for the original design you're replicating, not the one you've built.

**Comment by Tarush27** [comment]
Sorry for the wrong UI.
Below is the original UI.
Screenshot from 2022-01-08 18-46-01

**Comment by abhaysood** [comment]
The bell can simply be an emoji and be part of the text itself. No need of adding an icon in that case. Like this: 🔔 

Read more about unicode characters. Emojis are just unicode characters which text views in all frameworks can render without needing an image to be bundled with the app separately.

**Comment by Tarush27** [comment]
Ok @abhaysood I will retry to attach this bell icon along with my text view using  Unicode Characters.

**Comment by Tarush27** [comment]
@abhaysood I have now used a text view in which I have displayed the bell icon emoji through the use of string resource. So, through this, my query is resolved. I am attaching the UI for reference.
Screenshot from 2022-02-03 17-27-15

**Comment by Tarush27** [comment]
So, can we close the pull request now?

**Comment by abhaysood** [comment]
Sure

**Pull request merged**